features/promotion/applying_catalog_promotions/applying_catalog_promotions_for_taxon.feature

```diff
@@ -0,0 +1,36 @@
+@applying_catalog_promotions
+Feature: Applying catalog promotions for taxon
+    In order to be attracted to products
+    As a Visitor
+    I want to see discounted products in the catalog
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "Clothes" taxonomy
+        And the store has "Dishes" taxonomy
+        And the store has a "T-Shirt" configurable product
+        And this product main taxon should be "Clothes"
+        And this product has "PHP T-Shirt" variant priced at "$20.00"
+        And the store has a "Mug" configurable product
+        And this product main taxon should be "Clothes"
+        And this product has "PHP Mug" variant priced at "$5.00"
+        And there is a catalog promotion "Clothes sale" that reduces price by "30%" and applies on "Clothes" taxonomy
+
+    @todo
+    Scenario: Applying simple catalog promotions
+        When I view product "T-Shirt"
```

Comment:
We are adding a new rule for entire taxons so IMO you should have a taxon with 2 product and check if all of it has discounted price

Reply:
IMO next product is not needed. We have two products with two difference taxons. It doesn't matter if is 1 or 100 products under certain taxon.

Reply:
I would be for checking 2 variants from different products in `Clothes` taxonomy that have discounted prices, this will allow us to check if proper variants are provided by this rule